app/src/androidTest/java/com/github/lookupgroup27/lookup/ui/renderables/LabelTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.lookupgroup27.lookup.ui.renderables
```

Comment:
Thank you for the fix however, it seems the file is still not aligned with the package structure. Should it be moved to `com.github.lookupgroup27.lookup.model.map.renderables.label`? Or did I misunderstand the intended organization ?